pages/Albums.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react'
+import Page from '../components/Page'
+import Search from '../components/Search'
+import Album from '../components/Album'
+import Title from '../components/Title'
+import { filteredAlbums, retrieveAlbums } from '../utils/albums'
+
+export default () => {
+  const [albumList, setAlbumList] = useState([])
+  const [filteredAlbumList, setFilteredAlbumList] = useState([])
+  const [searchTerm, setSearchTerm] = useState('')
+
+  useEffect(() => {
+    retrieveAlbums(setAlbumList, setFilteredAlbumList)
+  }, [])
+
+  useEffect(() => {
+    filteredAlbums(albumList, searchTerm, setFilteredAlbumList)
+  }, [searchTerm])
```

Comment:
I've used this pattern of sending the setters to the util function as well. It becomes a little dirty looking when you get more setters but for this I agree it is cleaner